enterprise/config/keys/dev.pub.pem

```diff
@@ -1,9 +1,9 @@
 -----BEGIN PUBLIC KEY-----
-MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwOnNEhoNLnBiv9+Vw3NH
-o2hRZNp8eehBfJaoTdsF7GLYWZKcBjvSB/RuVdWdFJ2/X4QTkcJeKFN6q3w+86dY
-HKqrVzBxSxeIj1YMhyCNO4uECdrzBp8kcG7m8mN8r5glMMLE8ShAstPFCzKWbGw9
-aHFs3a5nQfWjn5X2SbfGLm1BhHzatDM6Cvmnt8NDSLqVjbPzT1TKH0owljYEIZwY
-m6BBHXyIQLmzsbb9yPVotYMn2cz4OUShwNW8LbXr8mjk+8lw3OcUn3W5CoF33aD4
-nsFFtgrlXmmgdaCP22bBha6iHz0Ff31lmuDZIYjaOAeTqyUxEV78Tg04ht6Kgmeu
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAqL2T+Q2U+XoShxOCLL6w
+Roe6+FgS6huDpqtkNjUz8xJlzusgoQdx5s5ucG1XMBXjFqmM/LKGO2tRgL0W/ywb
+yFPFaNkp096WRy0jIksrKCFvLtTKMqsjvV+NUtDLgC7CPShlAyPWKZgfBiNEIKcw
+8FEWynQxzfmb5Lh9iJzNohjm899n7urV3PcTnXhs9A7w24YyB1ykTjLnZssWS9h2
+EAsVkbj2yzWYCVWOmmO7s/LBFM+EUxcyYmkcB/y/sgGVMjo9z8F7v4ebO+Vk4t2E
+3RRrlseJFEZcNAfZoxt25Cl4VYWOj8LQeAOybG4l8Eb9lN4Xmic0T1gr+ABRcCKf
 owIDAQAB
```

Comment:
As console don't have local env, just remove this, also will update the pub/private keys in hub to align with console